packages/stream_chat_flutter/lib/src/message_list_view.dart

```diff
@@ -1130,6 +1072,21 @@ class _MessageListViewState extends State<MessageListView> {
       showPinButton: widget.pinPermissions.contains(currentUserMember.role),
     );
 
+    if (widget.messageBuilder != null) {
+      messageWidget = widget.messageBuilder!(
+        context,
+        MessageDetails(
+          userId,
+          message,
+          messages,
+          index,
+        ),
+        messages,
```

Comment:
two `messages` 😩

Reply:
yea but one is to populate the MessageDetails, the other one it's passed in the builder

Reply:
but not really needed as part of message details

Reply:
but we don't have any instance of the message list in the message details
it's just for computing the message details properties

Reply:
got it